src/pytraits/core/composing/compiler.py

```diff
@@ -40,7 +40,7 @@ def _clone_function(self, function):
 
         trait["co_argcount"] = function.__code__.co_argcount
         trait["co_kwonlyargcount"] = function.__code__.co_kwonlyargcount
-        if sys.version_info[0] == 3 and sys.version_info[1] >= 8:
+        if sys.version_info[:1] >= (3, 8) :
```

Comment:
must be  :2 sorry I thought I had fixed that

Reply:
Oh sorry, I thought the slice indices to be inclusive, which is apparently not true. (Your suggestion above was correct.)

Reply:
oh dear... this been here a while. Thank you a lot, I appreciate the effort! I stumbled similar error yesterday in different context.